Src/IronPython/Modules/sys.cs

```diff
@@ -406,12 +406,12 @@ public sealed class SysFlags : PythonTuple {
 
         #endregion
 
-        // These values are based on the .NET 2 BigInteger in Microsoft.Scripting.Math
-        public static longinfo long_info = new longinfo(32, 4);
+        // These values are based on the .NET BigInteger in System.Numerics.BigInteger
+        public static intinfo int_info = new intinfo(32, 4);
 
         [PythonType("long_info"), PythonHidden]
```

Comment:
```suggestion
        [PythonType("int_info"), PythonHidden]
```
This shows up as the type name: `type(sys.int_info)`

Reply:
Oops!